test/lib/marshal/email-layout/reader.test.ts

```diff
@@ -215,23 +214,23 @@ describe("lib/marshal/layout/reader", () => {
           updated_at: "2023-10-02T19:24:48.714630Z",
         });
       });
+    });
 
-      describe("with the withExtractedFiles opt of true", () => {
-        it("reads layout.json with the extracted fields inlined", async () => {
-          const [layout] = await readEmailLayoutDir(emailLayoutDirCtx, {
-            withExtractedFiles: true,
-          });
+    describe("with the withExtractedFiles opt of true", () => {
```

Comment:
No change here, but this test was nested for no reason so unnesting it resulted in diffs due to an indentation change.